lib/loader/loader.c

```diff
@@ -320,3 +324,133 @@ struct evmc_vm* evmc_load_and_configure(const char* config, enum evmc_loader_err
         evmc_destroy(vm);
     return NULL;
 }
+
+
+evmc_create_steppable_fn evmc_load_steppable(const char* filename,
```

Comment:
I guess it might be possible to factor out everything up to line 402 into a common helper function used by this and the `evmc_load` function to avoid code duplication, but I guess it is simpler to merge in upstream changes this way ..

Reply:
Yes, I am aware of the code duplication, but I didn't want to touch any existing code. I wanted this to be solely additional to what is already there, to make merging upstream changes easier.